bin/check_go_path

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ -z "$GOPATH" ]; then
+	echo "GOPATH not set, you must have go configured properly to install ipfs"
```

Comment:
Yes, but it doesn't come off the right way. You use them to divide up independent clauses, which ain't right.